alchemist-physics/src/test/resources/testCustomizeGlobalReactionRate3.yml

```diff
@@ -0,0 +1,9 @@
+incarnation: protelis
+
+environment:
+  type: EnvironmentWithDynamics
+  global-programs:
+    - time-distribution:
+        type: DiracComb
```

Comment:
Ok. Actually, I've also noticed something that seems strange to me.
This is the test log
```text
10:05:31.331 [pool-1-thread-1] INFO it.unibo.alchemist.loader.m2m.SimulationModel - Created environment: it.unibo.alchemist.model.implementations.environments.EnvironmentWithDynamics@45f5637c
10:05:31.336 [pool-1-thread-1] DEBUG it.unibo.alchemist.loader.m2m.SimulationModel - Trying to build a Pair using syntax {time-distribution={type=DiracComb, parameters=[0.5]}, type=PhysicsUpdate} from it.unibo.alchemist.loader.m2m.syntax.DocumentRoot$Environment$GlobalProgram@6154fdec
10:05:31.648 [pool-1-thread-1] DEBUG it.unibo.alchemist.loader.m2m.JVMConstructor - Building a DiracComb with [0.5]
10:05:31.671 [pool-1-thread-1] INFO it.unibo.alchemist.loader.m2m.JVMConstructor - Constructor DiracComb(Time, double) failed for the following reasons: 
10:05:31.672 [pool-1-thread-1] INFO it.unibo.alchemist.loader.m2m.JVMConstructor -   - InstancingImpossibleException: Could not create an instance with public it.unibo.alchemist.model.implementations.timedistributions.DiracComb(it.unibo.alchemist.model.interfaces.Time,double): discarded because incompatible with provided parameters [Double]
```
Looking at DiracComb implementation, there is another constructor that takes only the distribution rate, but is not evaluated

Reply:
Please fix this `toString`: `it.unibo.alchemist.loader.m2m.syntax.DocumentRoot$Environment$GlobalProgram@6154fdec`

The system should try both constructors, as it does in any other case, without a debugger it is hard to guess what's going on

Reply:
> Log also looks like this with other configuration where Dirac Comb is used

does this mean that you cannot create a normal reaction with a `DiracComb` specifying the rate? It should break a lot of tests

Reply:
Okay, but still fix the `toString` :smile:

Reply:
I see. Yes the shorter one is better